x-pack/plugins/cases/public/components/all_cases/use_cases_columns.tsx

```diff
@@ -169,12 +126,9 @@ export const useCasesColumns = ({
       field: 'assignees',
       name: i18n.ASSIGNEES,
       render: (assignees: Case['assignees']) => (
-        <AssigneesColumn
-          assignees={assignees}
-          userProfiles={userProfiles}
-          currentUserProfile={currentUserProfile}
-        />
+        <AssigneesColumn assignees={assignees} userProfiles={userProfiles} />
       ),
+      width: '20%',
```

Comment:
20% seems a bit high to me. Each avatar is around 25px and the button is around 80px. 10% should be enough.

Reply:
Checking it out now, but similar to what @cnasikas is saying, wonder if put a pixel value rather than a percentage.  I'll mess with it a bit, but a percentage seems tough to nail down and insure we avoid it breaking on two lines.

Reply:
Let's put the max avatars/assignees shown to 3 and set the width to either 20 percent or 180px.  

I'm not terribly concerned with the button getting pushed to the next row, but it would be nice to avoid it (the 180px width should avoid this).  We will want to assess the other columns as well (and consolidate them).  I have some thoughts on this which we can hopefully tackle in the near future.